dataland-frontend/tests/e2e/specs/landing-page/SamplePage.ts

```diff
@@ -2,8 +2,8 @@ import { getBaseUrl } from "@e2e/utils/Cypress";
 
```

Comment:
Let's talk about the changes here.  Since I changed this file the last time, I should be able to assist.